Big-Spender/readme.md

```diff
@@ -124,7 +136,17 @@ CREATE YOUR QUERY HERE
 **You:** Sure thing. To confirm, the date is August 19, 2021, the transaction number is 38104091, the supplier invoice number is 3780119655, the supplier is 'Dell', the expense type is 'Hardware' and the expense area is 'IT'. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERIES HERE
+insert into spends (expense_type_id, expense_area_id, supplier_id, date, transaction_no, supplier_inv_no, description, amount)
+values (
+    (select id from expense_types where expense_type = 'Hardware'),
+    (select id from expense_areas where expense_area = 'IT'),
+    (select id from suppliers where supplier = 'Dell'),
+    '2021-08-19',
+    38104091,
+    '3780119655',
+    'Computer Hardware Dell',
+    32000
+);
```

Comment:
Good attempt here. The rows ' Dell', 'Hardware' and 'IT' are missing from other tables, what extra step is needed to have the rows populated where necessary? What will this change have on the 'id's inserted in spends table?